src/react-app-env.d.ts

```diff
@@ -1 +1,45 @@
 /// <reference types="react-scripts" />
+
+export interface PostsFromServer {
+  userId: number,
+  id: number,
+  title: string;
+  body: string;
+}
+
+export interface CommentsFromServer {
```

Comment:
It's type for one comment
```suggestion
export interface CommentFromServer {
```